components/templates/src/lib.rs

```diff
@@ -5,6 +5,13 @@ extern crate tera;
 extern crate base64;
 extern crate pulldown_cmark;
 extern crate csv;
+extern crate crypto_hash;
+extern crate chrono;
+extern crate reqwest;
+extern crate url;
+
+#[macro_use]
+extern crate clap;
```

Comment:
why are we loading clap?

Reply:
See https://github.com/Keats/gutenberg/pull/494#discussion_r227265030